templates/MMM-NHL.njk

```diff
@@ -59,16 +59,63 @@
                     {% if config.showLogos %}
                         <td>
                             <img class="icon {{ "no-color" if not config.colored }}"
-                                 src="https://assets.nhle.com/logos/nhl/svg/{{ games[index].teams.away.short }}_dark.svg"/>
+                                 src="https://www-league.nhlstatic.com/images/logos/teams-20212022-dark/{{games[index].teams.away.id}}.svg"/>
                         </td>
                     {% endif %}
                     {% if config.showNames %}
                         <td class="align-left">
-                            {{ games[index].teams.away.short }}
+                            {{ games[index].teams.away.short if games[index].teams.away.short else games[index].teams.away.name  }}
                         </td>
                     {% endif %}
                 </tr>
             {% endfor %}
         </tbody>
     </table>
+    {% if config.showPlayoffSeries and playoffSeries and playoffSeries.length > 0 %}
+        <header style="margin-top: 10px">Series</header>
+        <table class="small table">
+            <thead>
```

Comment:
All header columns are empty. I don't think we need to have the `<thead>` at all.